src/components/limited-restoration-panel/LimitedRestorationPanel.stories.ts

```diff
@@ -14,26 +14,21 @@ const Template = (args, { argTypes }) => ({
 
 export const Default = Template.bind({})
 Default.args = {
-  currentDate: '2023-02-03',
-  expiryDate: null
+  months: 0
```

Comment:
As you can see, this component now takes simply the "months" value property, and it emits an updated months value (and valid value).

This is a big change so when I publish this then I will increase the major release number.

Reply:
Basically I changed this component to only care about the expiry months -- a simpler concern, and the parent app can handle more complicated logic (or in this case, simpler -- you'll see when you review my Edit UI changes).